src/platyPS/platyPS.psm1

```diff
@@ -2603,6 +2636,9 @@ function ConvertPsObjectsToMamlModel
                         # https://github.com/PowerShell/platyPS/issues/211
                         'Confirm' { "Prompts you for confirmation before running the cmdlet.`r`n`r`n" }
                         'WhatIf' { "Shows what would happen if the cmdlet runs. The cmdlet is not run.`r`n`r`n" }
+                        'IncludeTotalCount' { "Reports the number of objects in the data set (an integer) followed by the objects. If the cmdlet cannot determine the total count, it returns 'Unknown total count'.`r`n`r`n" }
+                        'Skip' { "Ignores the first 'n' objects and then gets the remaining objects.`r`n`r`n" }
+                        'First' { "Gets only the first 'n' objects.`r`n`r`n" }
```

Comment:
For Skip and First, can we re-use the `Select-Object` documentation about them? https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/select-object?view=powershell-6

`Skips (does not select) the specified number of items.`

`Gets only the specified number of objects. Enter the number of objects to get.`

Reply:
Sure, I duplicated what was in the official documentation as mentioned in the issue, but if you prefer otherwise, just let me know.
![image](https://user-images.githubusercontent.com/11078689/43299889-f80f9b56-9121-11e8-9e04-181c833be0ea.png)

Reply:
Gotcha, that's fine then, I dind't know about the supportsPaging and somehow the link slipped out from my attention. Let's keep the words from this doc then.